certbot/CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
-*
+* Fix nginx plguin crash when non-ASCII configuration file is being read
```

Comment:
should be "plugin"

Reply:
🙈 Fixed in 2ae090529e66a059e0e077d233f427583caa7c6c.